eng-wasm/src/lib.rs

```diff
@@ -123,5 +127,10 @@ mod tests {
     fn what() {
         print("TEST!");
     }
+
+    #[test]
+    fn test_encrypt() {
+        // TODO: Is this the right place to test APIs. If so, how should we initialize the enclave?
```

Comment:
It is better to put the tests [here](https://github.com/enigmampc/enigma-core/blob/develop/enigma-core/app/src/wasm_u/wasm.rs#L61)

1. Create the contract that uses encrypt/decrypt services.
2. Put the contract in https://github.com/enigmampc/enigma-core/tree/develop/examples/eng_wasm_contracts
3. **In Cargo.toml of the contract use the local paths for `eng-wasm` and `eng-wasm-derive`** instead of crates from crates.io.
4. Create a test in wasm.rs. All the helper functions are there as well as examples of working with `ethabi::Token`. Use the existing tests for reference. Here is the simplest form of the test https://github.com/enigmampc/enigma-core/blob/develop/enigma-core/app/src/wasm_u/wasm.rs#L363.

Reply:
Done.